web/WebDevelopersGuide.md

```diff
@@ -1188,6 +1190,11 @@ as needed.
  * `VariableDefinition created`
  * `VariableDefinition updated`
  * `VariableDefinition removed`
+
+##### Class `HeapInfo`
+ * `long maximumHeapSize` - Maximum heap size of this worker.
```

Comment:
these three should be number - that said, this document is not the canonical docs any more, should be in deephaven.io